terraform/graph_builder_apply.go

```diff
@@ -182,10 +182,13 @@ func (b *ApplyGraphBuilder) Steps() []GraphTransformer {
 			GraphTransformMulti(
 				&DestroyValueReferenceTransformer{},
 				&DestroyOutputTransformer{},
-				&PruneUnusedValuesTransformer{},
 			),
 		),
 
+		&PruneUnusedValuesTransformer{
```

Comment:
There's a stale comment on the `GraphTransformIf` above that still describes this as being a "destroy-time transformation'.